app/workers/codewars_recorder.rb

```diff
@@ -5,10 +5,13 @@ class CodewarsRecorder
   def perform(user_id, codewars_username)
     user = User.includes(:codewars).find(user_id)
     codewars = HTTParty.get("http://www.codewars.com/api/v1/users/#{codewars_username}/code-challenges/completed")['data']
-    codewars.each do |codewar|
-      codewar["completedLanguages"].each do |language|
-        unless user.codewars.where(:slug=>codewar["slug"], :language=> language).present?
-          user.codewars.create(:slug => codewar["slug"], :language => language)
+    # only check languages if we have a respons
+    if codewars
+      codewars.each do |codewar|
+        codewar["completedLanguages"].each do |language|
+          unless user.codewars.where(:slug=>codewar["slug"], :language=> language).present?
```

Comment:
Line is too long. [91/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.